src/main/java/uk/gov/hmcts/darts/common/datamanagement/helper/StorageOrderHelper.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmcts.darts.common.datamanagement.helper;
+
+import jakarta.annotation.PostConstruct;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import uk.gov.hmcts.darts.common.datamanagement.enums.DatastoreContainerType;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
```

Comment:
This annotation can be removed - not needed

Reply:
it is required, if removed it causes the application to fail to start:-
![image](https://github.com/hmcts/darts-api/assets/131666106/6b6dfa1a-e662-49fc-be46-0c5f6cf79d69)

Reply:
Normally it would be `@Component` if you want a bean in spring boot

Reply:
done